### ISSUE TITLE
Add share option to image context menu

### ISSUE THREAD

**Issue by NotWoods** [mozilla-mobile/android-components#3648]
[Originally filed](https://github.com/mozilla-mobile/fenix/issues/3621) by @BlackLotus:
> It's not possible to send/share media through the media menu.
> Examples 1:
> Graphics share menu only has save and send url, you can't share images out of the menu anymore. So right now you have to save, open and then share the image.
> ...
> 
> It would be nice to have at least feature parity with the mobile firefox.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-2995)

**Comment by pocmo** [comment]
This dropped of the list because that requires a "download" step before sharing and we didn't have anything for that in place. :)

**Comment by NotWoods** [comment]
Blocked by #3551

**Comment by davidwill91** [comment]
Is there any chance this will be added in the future?

**Comment by csadilek** [comment]
Moved to bugzilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1793138

Change performed by the [Move to Bugzilla add-on](https://addons.mozilla.org/en-US/firefox/addon/move-to-bugzilla/).
